polling_stations/apps/data_importers/management/commands/import_teignbridge.py

```diff
@@ -4,23 +4,15 @@
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "TEI"
     addresses_name = (
-        "2023-05-04/2023-03-10T11:50:22.113903/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-02-26T14:53:52.912490/Democracy_Club__02May2024.tsv"
     )
     stations_name = (
-        "2023-05-04/2023-03-10T11:50:22.113903/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-02-26T14:53:52.912490/Democracy_Club__02May2024.tsv"
     )
-    elections = ["2023-05-04"]
+    elections = ["2024-05-02"]
     csv_delimiter = "\t"
 
     def station_record_to_dict(self, record):
-        # Alice Cross Day Centre
-        if record.polling_place_id == "8273":
-            record = record._replace(polling_place_easting="293930")
-            record = record._replace(polling_place_northing="73047")
-
-        if record.polling_place_id == "8292":
-            record = record._replace(polling_place_postcode="")
-
         return super().station_record_to_dict(record)
```

Comment:
Can you remove the whole method please.

Reply:
Same synchronization issue as Melton but I have removed the method!